modules/Kconfig.cmsis_dsp

```diff
@@ -0,0 +1,513 @@
+# Copyright (c) 2020 Stephanos Ioannidis <root@stephanos.io>
+# SPDX-License-Identifier: Apache-2.0
+
+comment "Components"
+
+config CMSIS_DSP_BASICMATH
+	bool "Basic Math Functions"
+	default y
+
+config CMSIS_DSP_COMPLEXMATH
+	bool "Complex Math Functions"
+	imply CMSIS_DSP_FASTMATH
+
+config CMSIS_DSP_CONTROLLER
+	bool "Controller Functions"
+
+config CMSIS_DSP_FASTMATH
+	bool "Fast Math Functions"
+	select CMSIS_DSP_TABLES
+	imply CMSIS_DSP_TABLES_ALL_FAST
+
+config CMSIS_DSP_FILTERING
+	bool "Filtering Functions"
+
+config CMSIS_DSP_MATRIX
+	bool "Matrix Functions"
+
+config CMSIS_DSP_STATISTICS
+	bool "Statistics Functions"
+	imply CMSIS_DSP_BASICMATH
+	imply CMSIS_DSP_FASTMATH
+
+config CMSIS_DSP_SUPPORT
+	bool "Support Functions"
+
+config CMSIS_DSP_TRANSFORM
+	bool "Transform Functions"
+	select CMSIS_DSP_TABLES
+	imply CMSIS_DSP_TABLES_ALL_FFT
+
+config CMSIS_DSP_SVM
+	bool "Support Vector Machine Functions"
+
+config CMSIS_DSP_BAYES
+	bool "Bayesian Estimators"
+	imply CMSIS_DSP_STATISTICS
+
+config CMSIS_DSP_DISTANCE
+	bool "Distance Functions"
+	imply CMSIS_DSP_STATISTICS
+
+menuconfig CMSIS_DSP_TABLES
+	bool "Look-up Tables"
```

Comment:
I think it would be nice to the users of Kconfig, if we did provide a little help text on those settings.

Reply:
Agreed. I will add a brief explanation for these.

Reply:
Done